src/cpp/shared/town/town.h

```diff
@@ -27,7 +48,8 @@ enum FACTION
 	FACTION_MULTIPLE = 6,
 	FACTION_RANDOM = 7,
 
-	FACTION_CYBORG = 13,
+	FACTION_CYBORG = 12,
+	NUM_FACTIONS
```

Comment:
NUM_FACTIONS is now misnamed; it now actually represents 1 + the max faction ID (13), rather than the numebr of factions (7).